sdk/digitaltwins/azure-digitaltwins/azure/digitaltwins/_digitaltwin_models_client.py

```diff
@@ -0,0 +1,116 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ._generated import AzureDigitalTwinsAPI
+
+class DigitalTwinModelsClient(object):
+    """Creates an instance of AzureDigitalTwinsAPI.
+
+    :param endpoint: The URL endpoint of an Azure search service
+    :type endpoint: str
+    :param credential: A credential to authenticate requests to the service
+    :type credential: ~azure.core.credentials.AzureKeyCredential
+    :**kwargs Used to configure the service client.
+    :type **kwargs: any
+    """
+    def __init__(self, endpoint: str, credential: object, **kwargs: any):
+        # type: (str, AzureKeyCredential, **Any) -> None
+
+        self.endpoint = endpoint #type: str
+        self.credential = credential #type AzureKeyCredential
+        self._client = AzureDigitalTwinsAPI(
+            credential=credential,
+            base_url=endpoint,
+            **kwargs
+        ) #type: AzureDigitalTwinsAPI
+
+    def get_model(self, model_id, include_model_definition=False, **kwargs):
+        # type: (str, bool, **Any) -> "models.ModelData"
+        """Get a model, including the model metadata and the model definition
+
+        :param model_id: The Id of the model
+        :type model_id: str
+        :param include_model_definition: When true the model definition
+        will be returned as part of the result
+        :type include_model_definition: bool
+        :**kwargs The operation options
+        :type **kwargs: any
+        :returns: The ModelDate object
+        :rtype: ~azure.digitaltwins.models.ModelData
+        :raises :class: `~azure.core.exceptions.HttpResponseError`
+        """
+        return self._client.digital_twin_models.get_by_id(model_id, include_model_definition, **kwargs)
+
+    def list_models(self, dependencies_for, include_model_definition=False, max_item_count=-1, **kwargs):
```

Comment:
I'm not sure about the name `dependencies_for` - is this what we're using in other languages?
The docstring implies this can be multiple "models Id", but the parameter is a single string?
The two parameters `include_model_definition` and `max_item_count` should be made keyword only, and rolled into `kwargs`.

Reply:
Yes the dependencies_for is the name we are uding across languages. But the type was wrong, it will be list[str] to provide list of model IDs.
Fixed.

---
In reply to: [491745328](https://github.com/Azure/azure-sdk-for-python/pull/13852#discussion_r491745328) [](ancestors = 491745328)